lib/fortnox/api/types/order_row.rb

```diff
@@ -9,7 +9,7 @@ class OrderRow < DocumentRow
         STUB = { ordered_quantity: 0 }.freeze
 
         # OrderedQuantity Ordered quantity
-        attribute :ordered_quantity, Types::Required::Float
+        attribute :ordered_quantity, ::Fortnox::API::Types::Required::Float
```

Comment:
Is the prefix needed? Shouldn't be ...

Reply:
I removed it and it still worked, so let's stick with just `Types::...` 😄 Don't know why this was changed in the first place.